GeoLib/GEOObjects.cpp

```diff
@@ -90,29 +91,21 @@ bool GEOObjects::appendPoint(Point* point, std::string const &name, std::size_t&
 const std::vector<Point*>* GEOObjects::getPointVec(const std::string &name) const
 {
 	int idx = this->exists(name);
```

Comment:
we can do this in a seperate PR. the problem is, that you are introducing quite a few casts now (basically everywhere where exists() is used) and I would assume that it is easier to track the changes now.
the actual difference would only be the return value and type of ```exists()``` and changing the test ```if (i>0)```  to ```if (i<std::numeric_limits<std:size_t>::max())```